notebooks/251-tiny-sd-image-generation/251-tiny-sd-image-generation.ipynb

```diff
@@ -54,7 +54,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        with open("unet_calibration_example_input.pkl", "rb") as f:
</pre><p>we prefer do not store data files in the repo, can we upload data in some another place e.g.  &nbsp;<a href="https://storage.openvinotoolkit.org" target="_blank">https://storage.openvinotoolkit.org</a> will be better place for that?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Uploaded to https://storage.openvinotoolkit.org/repositories/openvino_notebooks/data/data/pkl/unet_calibration_example_input.pkl